### PR TITLE
Clean up format and style, prepare for Databricks runtime

### PR DESCRIPTION
**Pull request by henrikbulldog**

### CODE REVIEW

**Review by Rajesh2015 — APPROVED** [review]
LGTM

**Comment by Rajesh2015** [comment]
@henrikbulldog I see the sparksession is never closed in test class after the test ends though I donot see any side effects  .is it intentional .

**Review by sudhanshurocky — APPROVED** [review]

File: src/test/scala/org/spark/NameConverterTest.scala [hunk 1]
```diff
@@ -33,55 +35,129 @@ class NameConverterTest extends FlatSpec {
 
   import spark.implicits._
 
+  val firstNames = Source.fromFile("src/test/scala/org/spark/data/firstNames.txt").getLines.toSeq
```
Comment (Rajesh2015) [inline src/test/scala/org/spark/NameConverterTest.scala:38]: @henrikbulldog Test cases are failing as the txt file name has discrepancy (Uppercase Lower Case)

File: src/test/scala/org/spark/NameConverterTest.scala [hunk 2]
```diff
@@ -33,55 +35,129 @@ class NameConverterTest extends FlatSpec {
...
+  val lastNames = Source.fromFile("src/test/scala/org/spark/data/lastNames.txt").getLines.toSeq
```
Comment (Rajesh2015) [inline src/test/scala/org/spark/NameConverterTest.scala:39]: Same as previous

**Review by Rajesh2015 — CHANGES_REQUESTED** [review]
test data names are having issue

**Review by Rajesh2015 — APPROVED** [review]
LGTM

**Review by sudhanshurocky — APPROVED** [review]

**Pull request merged**